src/main/theme/operating-system-theme.injectable.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { computed } from "mobx";
+import operatingSystemThemeStateInjectable from "./operating-system-theme-state.injectable";
+
+const operatingSystemThemeInjectable = getInjectable({
```

Comment:
Because we shouldn't expose the `.set(` at all. Since there is no way to actually set it. 

In future, we could introduce possibility to add "private" injectables which cannot be injected outside of given scope (e.g file). This would remove requirement for these type of injectables. But that's for the future. :)